karma.conf.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function (config) {
+	config.set({
+		frameworks: ['ava'],
+		files: [
+			'test/shortcut-*.js',
+			'test/zen-provided.js'
+		],
+		browsers: [process.env.CI ? 'PhantomJS' : 'Chrome'],
```

Comment:
We could probably have `karma-ava` do that. Not sure if the karma team would go for that as a karma default (you could always open an issue).